src/helpers/array-helpers.js

```diff
@@ -1,6 +1,6 @@
 export function reject(array, callback) {
```

Comment:
please add a default parameter instead

Reply:
Fixed. Happy to see you've joined as a contributor 👍 .

Reply:
Can you also rebase and fixup the commits to have only 1 commit in the PR?